libs/sdk-model/src/execution/executionDefinition/factory.ts

```diff
@@ -70,6 +71,8 @@ export function newDefForItems(
         measures: items.filter(isMeasure),
     };
 
+    validateExecDefinition(def);
```

Comment:
i suggest to rename this to match conventions in this area.. call it something like `defValidate` (e.g. we have `defFingerprint` already)

Reply:
Good point, updated